src/main/java/nl/utwente/presto/tezos/tezos/Block.java

```diff
@@ -89,4 +90,21 @@
     private long nOpsImplicit;
     private boolean lbEscVote;
     private long lbEscEma;
+
+    public Block getBlock() {
+        return this;
+    }
+
+    public Number getSize() { // TODO check if this is what they want from getSize
```

Comment:
Volgens mij wordt dit alleen gebruikt om de progress van de query te laten zien (hoeveel data er al is gedownload). Dus als je dat helemaal accuraat zou willen hebben dan zou je hier de grootte van de JSON response moeten returnen.

Reply:
Maar dat wordt in die Ethereum connector volgens mij ook niet gedaan

Reply:
Laten bij een TODO lijkt me dan goed voor nu

Reply:
Mij ook